### ISSUE TITLE
should pin pip

### ISSUE THREAD

**Issue by bolliger32** [RhodiumGroup/docker_images#52]
the pip version installed with the current notebook image is 9.0.3. It gives an error when trying to use it, until you update it to a newer version:

```
Traceback (most recent call last):
  File "/opt/conda/bin/pip", line 6, in <module>
    sys.exit(pip.main())
AttributeError: module 'pip' has no attribute 'main'
```

I think we should add pip to the conda install package line in the Dockerfile. The current version is 18.0 so we could pin to that version (on both worker and notebook)

**Comment by delgadom** [comment]
@bolliger32 how did you update pip? `conda update` or `pip upgrade`?

yeah it looks like conda-forge has pip 18.0. not sure how 9.0.3 always gets installed. maybe one of our dependencies needs it... I'll give it a whirl and see what happens.

Thanks

**Comment by delgadom** [comment]
closed by #65
